tests/Stubs/TransitionableModel.php

```diff
@@ -17,8 +17,8 @@ class TransitionableModel extends Model
     public bool $isTransitionedByCustomWorkflow = false;
     public int $timesTransitionedByQueuedWorkflow = 0;
 
-    /** @var array<string, class-string<\LenderSpender\LaravelEnums\Enum>> */
-    protected $enums = [
+    /** @var string[] */
```

Comment:
This is more accurate:

```suggestion
    /** @var array<string, class-string<\BackedEnum>> */
```